acme/acme/client_test.py

```diff
@@ -621,6 +622,26 @@ def test_requests_error_passthrough(self, mock_requests):
         self.assertRaises(requests.exceptions.RequestException,
                           self.net._send_request, 'GET', 'uri')
 
+    def test_urllib_error(self):
+        try:
+            # pylint: disable=protected-access
+            self.net._send_request('GET', "http://localhost:19123/nonexistent.txt")
+
+        except ValueError as y:
+            if "linux" in sys.platform:
+                self.assertEqual("Requesting localhost/nonexistent: "
+                                 "[Errno 111] Connection refused", str(y))
```

Comment:
Looking at the test makes me more convinced we should leave it out. Splitting up the test based on platform is pretty hacky, especially when we can simplify the error message and the test at the same time without removing useful information.